### DISCUSSION TITLE
Is anyone else noticing compilation bugs on CodeSandbox?

### DISCUSSION THREAD

**Discussion by brainkim** [bikeshaving/crank#92] [General]
I usually develop with a parcel 1 bundler in local development, but every once in a while I encounter odd bugs when I try to publish these local examples to CodeSandbox. In other words, the code runs fine in local development, but on CodeSandbox I’m encountering bugs related to `this` not being iterated over or `for ({value} of this)` not properly assigning variables (it depends on the variable name???). I haven’t done due diligence to try to narrow these bugs, but my best guess is that these bugs are not the result of anything internal to Crank.js but something hinky going on in the CodeSandbox babel/typescript toolchain when dealing with Crank idioms, which are admittedly new. Honestly though, I have no way to figure out what exactly is happening. I also notice that when people try to export CodeSandbox examples to play with them locally, they get `regeneratorRuntime` errors.

The worry is that people will blame these bugs on Crank.js, when really they have nothing to do with it. I wonder if switching to glitch or something would be better. It would be a real shame because the CodeSandbox founder was very kind and even featured Crank in the examples page, but these bugs are scary af.

**Comment by ryhinchey** [comment]
I haven’t had an issue running crank in codesandbox but I haven’t downloaded examples or uploaded new ones.

Glitch might not be a bad idea.  We could probably use webpack with a simple and transparent Babel setup
